src/app/visual/list-view-grid/list-view-grid-visual.component.html

```diff
@@ -7,7 +7,10 @@
       </sky-list-column-selector-action>
     </sky-list-toolbar>
 
-    <sky-list-view-grid fit="scroll" #grid>
+    <sky-list-view-grid
+      fit="scroll"
+      #grid
```

Comment:
Think we could add a visual test to demonstrate the highlighted style, just to make sure its getting translated through?

Reply:
done